exercises/card.js

```diff
@@ -14,27 +14,53 @@ class Card{
 
 class Deck{
 	constructor(){
+		this.cards = [];
 		//takes in nothing
 		//make storage for your card objects
 	}
 	addCard( suit, faceValue ){
+		var newCard = new Card(suit,faceValue);
+		this.cards.push(newCard);
+		return this.cards.length;
```

Comment:
Technically the push method also returns the current length after the push, could have returned that.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push

``` 
return this.cards.push(newCard)
```